### PR TITLE
Initial implementation of terraform-aws-kms-cmk

### PR DESCRIPTION
**Pull request by andysaputra**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+========================================
+
+Terraform module which creates a KMS Customer Master Key (CMK) and its alias.
+
+Usage
+-----
+
+```hcl
+module "cmk_key" {
+  source                  = "github.com/traveloka/terraform-aws-kms-cmk?ref=master"
```
Comment (sshvetsov) [inline README.md]: One should always use explicit version tag, not a good example of usage, please correct.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+  product_domain          = "bei"
+  alias_name              = "tvlk-key"
+  environment             = "prod"
```
Comment (sshvetsov) [inline README.md]: Wrong value for environment, please correct

File: main.tf [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+  key_usage               = "ENCRYPT_DECRYPT"
+  policy                  = "${var.key_policy}"
+  deletion_window_in_days = "${var.deletion_window_in_days}"
+  is_enabled              = true
+  enable_key_rotation     = true
+
+  tags {
+    Description   = "${var.description}"
+    Environment   = "${var.environment}"
+    AliasName     = "${var.alias_name}"
```
Comment (sshvetsov) [inline main.tf]: Should be *Name*

File: outputs.tf [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+
+output "key_arn" {
+  description = "The arn of the key"
+  value       = "${aws_kms_key.key.arn}"
+}
+
+output "key_id" {
+  description = "The globally unique identifier for the key"
+  value       = "${aws_kms_key.key.id}"
+}
```
Comment (sshvetsov) [inline outputs.tf]: Good idea to add an Alias as output as well.

File: examples/complete/main.tf [hunk 5]
```diff
@@ -0,0 +1,30 @@
+provider "aws" {
+  region = "ap-southeast-1"
+}
+
+data "aws_caller_identity" "current" {}
+
+data "aws_iam_policy_document" "cmk_key_policy" {
```
Comment (sshvetsov) [inline examples/complete/main.tf]: This policy is quite permissive. Is there a way to provide a custom policy to the CMK?

Comment (andysaputra) [reply examples/complete/main.tf]: That's just for example.

File: outputs.tf (hunk 4 above)
Comment (andysaputra) [reply outputs.tf]: done

**Pull request merged**